core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -26,10 +27,27 @@
 
 <style>
 
-  md-progress-linear.md-default-theme .md-bar {
+  .audio-bar md-slider {
```

Comment:
Actually I meant `audio-bar`, sorry for the confusion. This element is in a custom audio-bar tag so the CSS selector should be `audio-bar md-slider`. Ditto for the others.

And after that you can remove the audio-bar class from the HTML element.